src/test/java/io/carbynestack/cli/client/ephemeral/EphemeralClientCliTest.java

```diff
@@ -153,6 +153,31 @@ public void givenInputOptionWithInvalidUUID_whenParsing_thenExceptionWithCorrect
     }
   }
 
+  @Test
+  public void
+      givenBothTagFiltersAndInputsProvided_whenUsingEphemeralClient_thenExceptionWithCorrectMessageIsThrows()
+          throws CsCliLoginException, CsCliException {
+    String uuid = UUID.randomUUID().toString();
+    String tagFilter = "key:value";
+
+    try {
```

Comment:
I am aware that the existing tests for exceptions were all implemented with try/catch, but I would prefer the transition to Assert.assertThrows. Like

```java
CsCliRunnerException actualCCRE = Assert.assertThrows(CsCliRunnerException.class, () ->
      getCliWithArgs(
              ExecuteEphemeralClientCliCommandConfig.COMMAND_NAME,
              "-i",
              uuid,
              "-f",
              tagFilter,
              "app.example.com")
          .parse());
    assertEquals(actualCCRE.getMessage(),
            EPHEMERAL_CLI_MESSAGES.getString("execute.failure.both-inputs-provided"));
```